mplbasketball/court3d.py

```diff
@@ -631,51 +676,51 @@ def draw_court_3d(
     # hoop: the hoop
     # x-offsets are for mirroring about x
     # z-offsets for verticality
-    bb_xoffset = court.court_parameters['backboard_distance_from_edge'] - court.court_parameters['court_dims'][0]/2
-    bb_zoffset = court.court_parameters['hoop_height'] - court.court_parameters['backboard_inner_rect_from_bottom']
-    ss_zoffset = court.court_parameters['hoop_height']
-    hoop_xoffset = court.court_parameters['hoop_distance_from_edge'] - court.court_parameters['court_dims'][0]/2
-    hoop_zoffset = court.court_parameters['hoop_height']
+    bb_xoffset = court.court_parameters["backboard_distance_from_edge"] - court.court_parameters["court_dims"][0] / 2
+    bb_zoffset = court.court_parameters["hoop_height"] - court.court_parameters["backboard_inner_rect_from_bottom"]
+    ss_zoffset = court.court_parameters["hoop_height"]
+    hoop_xoffset = court.court_parameters["hoop_distance_from_edge"] - court.court_parameters["court_dims"][0] / 2
+    hoop_zoffset = court.court_parameters["hoop_height"]
     origin_shift_x, origin_shift_y = -court.origin
     court_x, court_y = court.court_parameters["court_dims"]
 
-
     # helper functions for building the hoop geometry
     # vrectangle_pts: creates a closed (meaning the last point is
     # a duplicate of the first point) rectangle with given height and width
     # where it is symmetric about the x-axis, and situated above the
     # xy plane
-    vrectangle_pts = lambda w, h: np.array([
-        [origin_shift_x, origin_shift_y - w/2, 0.0],
-        [origin_shift_x, origin_shift_y - w/2, h],
-        [origin_shift_x, origin_shift_y + w/2, h],
-        [origin_shift_x, origin_shift_y + w/2, 0.0],
-        [origin_shift_x, origin_shift_y - w/2, 0.0]
-    ]).T
+    def vrectangle_pts(w: float | int, h: float | int):
+        return np.array(
+            [
+                [origin_shift_x, origin_shift_y - w / 2, 0.0],
+                [origin_shift_x, origin_shift_y - w / 2, h],
+                [origin_shift_x, origin_shift_y + w / 2, h],
+                [origin_shift_x, origin_shift_y + w / 2, 0.0],
+                [origin_shift_x, origin_shift_y - w / 2, 0.0],
+            ]
+        ).T
 
     # hcircle_pts generates points to create a horizontal circle
-    hcircle_pts = lambda radius, n: np.array([
-        origin_shift_x + radius * np.cos(np.linspace(0, 2*np.pi, n)),
-        origin_shift_y + radius * np.sin(np.linspace(0, 2*np.pi, n)),
-        np.zeros(n)
-    ])
```

Comment:
Per **PEP 8**:  _"Always use a def statement instead of an assignment statement that binds a lambda expression directly to an identifier."_
[READ MORE HERE](https://docs.astral.sh/ruff/rules/lambda-assignment/)